src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -191,17 +191,45 @@ public static class StandardLearnersCatalog
         }
 
         /// <summary>
-        /// Predict a target using a linear binary classification model trained with the AveragedPerceptron trainer.
+        /// Predict a target using a linear binary classification model trained with averaged perceptron trainer.
         /// </summary>
+        /// <remarks>
+        /// Perceptron is a classification algorithm that makes its predictions based on a linear function.
+        /// For instance with feature values f0, f1,..., f_D-1, the prediction is given by the sign of sigma[0, D-1] (w_i * f_i), where w_0, w_1,..., w_D-1 are the weights computed by the algorithm.
+        ///
+        /// Perceptron is an online algorithm, i.e., it processes the instances in the training set one at a time.
```

Comment:
>Perc [](start = 12, length = 4)

will want to wrap in <para> to create the paragraphs.

Reply:
&lt;para&gt; -- xml tag missing from comment on github page.

---
In reply to: [255266758](https://github.com/dotnet/machinelearning/pull/2483#discussion_r255266758) [](ancestors = 255266758)

Reply:
I'm running docfx locally and the text turns out fine as is. I think the extra /// will serve as <para> 

I'll review it in smoke-test to make sure it looks ok.

---
In reply to: [255269520](https://github.com/dotnet/machinelearning/pull/2483#discussion_r255269520) [](ancestors = 255269520,255266758)